packages/platform-android/src/commands/runAndroid/index.ts

```diff
@@ -359,6 +366,7 @@ function startServerInNewWindow(
   logger.error(
     `Cannot start the packager. Unknown platform ${process.platform}`,
```

Comment:
Shouldn't we throw here? 

TIP: This codeebase hasn't been touched in a while, I am pretty sure it has some wild `logger.error` instead of `CLIError` in few places.

Good opportunity to update.

Reply:
It's not related to the PR, we can take it separately under "improving error messags and handling inside `run-android`". Whatever makes more sense!